cmd/ygg/main.go

```diff
@@ -154,8 +154,58 @@ func main() {
 		{
 			Name:  "status",
 			Usage: "reports status of connection and activation",
+			Flags: []cli.Flag{
+				// TODO: load this value from config ? (or have a default)
+				&cli.StringSliceFlag{
```

Comment:
This won't be necessary if we are only reporting on the status of the systemd service.